doc/topics/tutorials/states_pt1.rst

```diff
@@ -125,13 +125,16 @@ Next, let's run the state we created. Open a terminal on the master and run:
 
 .. code-block:: bash
 
-    % salt '*' state.highstate
+    salt '*' state.apply
 
-Our master is instructing all targeted minions to run :func:`state.highstate
-<salt.modules.state.highstate>`. When a minion executes a highstate call it
-will download the :ref:`top file <states-top>` and attempt to match the
-expressions. When it does match an expression the modules listed for it will be
-downloaded, compiled, and executed.
+Our master is instructing all targeted minions to run :func:`state.apply
+<salt.modules.state.apply>`. When this function is executied without any SLS
+targets, a minion will download the :ref:`top file <states-top>` and attempt to
+match the expressions within it. When the minion does match an expression the
+modules listed for it will be downloaded, compiled, and executed.
+
+.. note::
+    This action is referred to as a "highstate".
```

Comment:
I think we need to expand this documentation to include an explanation stating that `state.apply` is what was previously known as `state.highstate` and has been aliased to make it easier to understand.